cpp/oneapi/dal/backend/primitives/objective_function/logloss.hpp

```diff
@@ -104,13 +104,13 @@ sycl::event compute_raw_hessian(sycl::queue& q,
                                 const event_vector& deps = {});
 
 template <typename Float>
-class LogLossHessianProduct : public BaseMatrixOperator<Float> {
+class logloss_hessian_product : public base_matrix_operator<Float> {
```

Comment:
Is it approved naming change?

Reply:
Yes, it is internal primitve which is not used by other algorithms. Naming should be alligned with snake_case used in out library